src/VisualStudio/IntegrationTest/TestUtilities/VisualStudioInstanceFactory.cs

```diff
@@ -332,6 +334,10 @@ private static Process StartNewVisualStudioProcess(string installationPath, int
                 // Disable background download UI to avoid toasts
                 Process.Start(CreateSilentStartInfo(vsRegEditExeFile, $"set \"{installationPath}\" {Settings.Default.VsRootSuffix} HKCU \"FeatureFlags\\Setup\\BackgroundDownload\" Value dword 0")).WaitForExit();
 
+                var lspRegistryValue = usingLspEditor ? "1" : "0";
+                Process.Start(CreateSilentStartInfo(vsRegEditExeFile, $"set \"{installationPath}\" {Settings.Default.VsRootSuffix} HKCU \"FeatureFlags\\Roslyn\\LSP\\Editor\" Value dword {lspRegistryValue}")).WaitForExit();
```

Comment:
Any way this can reuse the constant defining the Roslyn value elsewhere? Not sure if that's possible due to layering so this might be a bad thing you shouldn't even try to do.